packages/2d/src/components/Video.ts

```diff
@@ -263,11 +284,39 @@ export class Video extends Rect {
     }
   }
 
+  protected setPlaybackRate(playbackRate: number) {
+    let value: number;
+    if (isReactive(playbackRate)) {
+      value = playbackRate();
+      useLogger().warn({
+        message: 'Invalid value set as the playback rate',
+        remarks: reactivePlaybackRate,
+        inspect: this.key,
```

Comment:
Forgot about this but we should also add a stack trace here:
```suggestion
        inspect: this.key,
        stack: new Error().stack,
```